index.html

```diff
@@ -0,0 +1,168 @@
+        <p>Sunzi Bistro is a well-established restaurant that has been in business for over 10 years. You can easily find this restaurant at the Remedios Circle in Malate because of its wooden door and green roof.</p>
+    </header>
+    <div class="features-bar">
+            <div class="dropdown1">
+                <button class="dropbtn1">Categories
+                  <i class="fa fa-caret-down"></i>
+                </button>
+                <div class="dropdown-categories">
+                    <a href="#">Hamburger</a>
+                    <a href="#">Noodles</a>
+                  <a href="#">Pasta</a>
+                  <a href="#">Pizza</a>
+                  <a href="#">Sweets</a>
+                </div>
+            </div>
+                <div class="dropdown2">
+                    <button class="dropbtn2">Sorting
+                      <i class="fa fa-caret-down"></i>
+                    </button>
+                    <div class="dropdown-sorting">
+                      <a href="#">Sort</a>
+                      
+                    </div>
+                </div>
+                    <div class="dropdown3">
+                        <button class="dropbtn3">Vegetarian
+                          <i class="fa fa-caret-down"></i>
+                        </button>
+                        <div class="dropdown-vegetarian">
+                          <a href="#">Noodlles</a>
+                        </div>
+                </div>
+        <div class="min-max-slider" data-legendnum="2">
+            <label for="min">Minimum price</label>
+            <input id="min" class="min" name="min" type="range" step="1" min="0" max="150" />
+            <label for="max">Maximum price</label>
+            <input id="max" class="max" name="max" type="range" step="1" min="0" max="150" />
+            <h6>Price</h6>
+        </div>
+      </div>
+      <br>
+      <section class="hero">
+        <div class="hero-content">
+          <a href="#" id="button" class="button2">&nbsp;+&nbsp;</a>
+        </div>
+      </section>
+      <div class="bg-modal">
+        <div class="modal-contents">
+          <div class="close">+</div>
+          <img class="logo2" src="img/logo.jpg" alt="">
+          <form action="">
+            <input class="modal" id="photo" type="text" placeholder="Product Photo">
+            <input class="modal" id="name" type="text" placeholder="Product Name">
+            <input class="modal" id="category" type="text" placeholder="Product Category">
+            <input class="modal" id="price" type="text" placeholder="Product Price">
+            <br>
+            <a href="#" class="button" id="btnAdd">Add</a>
+          </form>
+        </div>
+      </div>
+      <table class="image-table">
+        <tbody>
+          <tr class="statistics">
+          <td>
+            <img class="image" onmouseover="bigImg(this)" onmouseout="normalImg(this)" src="img/Pizza Salami.webp" alt="Pizza Salami" title="Pizza Salami">
```

Comment:
Pretty hard to read such a long line. When you have so many attributes you can put 1 bellow another.
<img class="image" 
         onmouseover="bigImg(this)"
....etc>